DSCResources/MSFT_xDBPackage/MSFT_xDBPackage.psm1

```diff
@@ -109,7 +109,7 @@ function Test-TargetResource
 
     $connectionString = Construct-ConnectionString -sqlServer $SqlServer -credentials $Credentials
 
-    $dbExists = Check-IfDbExists -databaseName $DatabaseName -connectionString $connectionString
+    $dbExists = CheckIfDbExists -connectionString $connectionString -databaseName $DatabaseName 
```

Comment:
Ultimately no it's not important. The order was simply changed so that it matched the order in which they were defined in the function that it was changed to (CheckIfDbExists instead of Check-IfDbExists).